helm-chart/eoapi/values.schema.json

```diff
@@ -108,6 +108,7 @@
         }
       }
     },
+
```

Comment:
Should `overrideRootPath` be in the other services schema entries?

Reply:
I think it is fine to have it only in the api services (`stac`, `vector`, `raster`, `multidim`)